apps/balancer-tools/src/app/apr/rounds/[roundId]/route.ts

```diff
@@ -1,31 +1,40 @@
 import { NextRequest, NextResponse } from "next/server";
 
-import { getBalEmissions } from "#/app/apr/(utils)/getBalEmission";
 import { getBALPriceByRound } from "#/app/apr/(utils)/getBALPriceByRound";
 import { Round } from "#/app/apr/(utils)/rounds";
+import * as balEmissions from "#/lib/balancer/emissions";
+import { Pool } from "#/lib/balancer/gauges";
 
 // import { DuneAPI } from "#/lib/dune";
 import { mockGetTVLByRoundId, voteGaugeByID } from "../../mock_apis";
 
 export async function GET(
   _: NextRequest,
-  context: { params: { roundId: number } },
+  context: { params: { roundId: number }; query: { poolId: string } },
```

Comment:
This is supposed to be a queryparam, right?
https://nextjs.org/docs/app/api-reference/functions/next-request#nexturl